Source/Scene/Cesium3DTileFeature.js

```diff
@@ -0,0 +1,262 @@
+/*global define*/
+define([
+        '../Core/Color',
+        '../Core/defineProperties'
+    ], function(
+        Color,
+        defineProperties) {
+    'use strict';
+
+    /**
+     * Provides access to a feature's properties stored in the 3D tile's batch table, as well
+     * as the ability to show/hide a feature and change its highlight color via
+     * {@link Cesium3DTileFeature#show} and {@link Cesium3DTileFeature#color}, respectively.
+     * <p>
+     * Modifications to a <code>Cesium3DTileFeature</code> object have the lifetime of the tile's
+     * content.  If the tile's content is unloaded, e.g., due to it going out of view and needing
+     * to free space in the cache for visible tiles, listen to the DOC_TBA event to save any
+     * modifications.
+     * </p>
+     * <p>
+     * Do not construct this directly.  Access it through {@link Cesium3DTileContent#getFeature}
+     * or picking using {@link Scene#pick} and {@link Scene#pickPosition}.
```

Comment:
I added more doc and an example for `pick`, but I'm not sure what to change about `pickPosition`.